lit-element/src/components/App.ts

```diff
@@ -6,17 +6,17 @@
  *
  */
 
-import { agentxJsApi } from "@agentx/agentx-js-api";
+import { Desktop } from "@wxcc-desktop/sdk";
 import { html, LitElement, customElement, internalProperty } from "lit-element";
 import styles from "./App.scss";
 import { logger } from "./sdk";
-import { Service } from "@agentx/agentx-services-types";
+import { Service } from "@wxcc-desktop/sdk-types";
```

Comment:
so Service type is available here as well?

Reply:
Yes and as sd-types contain both agentx-types and service-types, it takes care of it